src/cider/tasks.clj

```diff
@@ -0,0 +1,17 @@
+(ns cider.tasks
+  (:require [boot.core :refer [deftask]]
+            [boot.repl :as repl]
+            [boot.util :as util]))
+
+(deftask add-middleware
+  "CIDER middleware injection task
+
+  This task allows to inject middleware in boot.repl/*default-middleware*.
+  Just pass them as -m|-middleware. The input is a string but will be converted
+  to symbol."
+  [m middleware MW #{str} "String of the middleware to inject"]
+  (if-let [default-middleware (resolve 'boot.repl/*default-middleware*)]
+    (do (swap! @default-middleware concat (map symbol middleware))
+        (util/dbug "Injected middleware: %s\n" (vec @@default-middleware)))
```

Comment:
Yes, default is what boot injected and plus we show what we injected, maybe we can actually have two different log calls here.